src/ImageSharp/ImageFrame{TPixel}.cs

```diff
@@ -342,8 +341,7 @@ protected override void Dispose(bool disposing)
 
         if (disposing)
         {
-            this.PixelBuffer?.Dispose();
-            this.PixelBuffer = null;
```

Comment:
There's a test to check if it is null.

Is that check mandatory?

Reply:
Let me check

Reply:
No. the pixel buffer is always assigned.